### PR TITLE
feat: restore ESM functionality

### PR DESCRIPTION
**Pull request by RichAyotte**
Improves ESM compatibility

refs: https://github.com/ExodusMovement/assets/pull/2449

### CODE REVIEW

**Review by feri42 — APPROVED** [review]
utACK

File: package.json [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+    "module": "./src/index.js",
     "react-native": {
         "./lib/browser.cjs": "./lib/native.cjs"
     },
     "exports": {
         "./package.json": "./package.json",
         ".": {
-            "require": "./lib/browser.cjs"
+            "require": "./lib/browser.cjs",
+            "import": "./src/index.js"
```
Comment (ChALkeR) [inline package.json:15]: This is not correct, we should never use `index.js` entry point, that's a different implementation

Comment (ChALkeR) [reply package.json:15]: Should be `./src/browser.js` perhaps

File: package.json [hunk 2]
```diff
@@ -1,16 +1,18 @@
 {
     "name": "@exodus/hashgraph-sdk",
-    "version": "2.6.0-exodus.11",
+    "version": "2.6.0-exodus.12",
     "description": "Hedera™ Hashgraph SDK",
     "types": "./lib/index.d.ts",
-    "main": "./lib/browser.cjs",
+    "main": "../src/index.js",
```
Comment (ChALkeR) [inline package.json:6]: This is incorrect

**Comment by ChALkeR** [comment]
nACK

**Comment by ChALkeR** [comment]
Replaced with 4d095619affd060b9f2457754e8366da964e4ab3

**Pull request merged**